airflow/providers/cncf/kubernetes/example_dags/example_spark_kubernetes_operator_spark_pi.yaml

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
Can you add this file to MANIFEST.in? 
https://github.com/apache/airflow/blob/master/MANIFEST.in

Reply:
done